src/HotChocolate/Core/src/Types.OffsetPagination/CollectionSegment.cs

```diff
@@ -31,7 +31,8 @@ public class CollectionSegment : IPage
         {
             Items = items ??
                 throw new ArgumentNullException(nameof(items));
-            Info = info;
+            Info = info ??
+                throw new ArgumentNullException(nameof(info));
```

Comment:
This wasn't here before, but I think it should've.

Reply:
Was I think also a mistake that it was not there.

Reply:
It's declared as non-nullable in the schema, so I think it's the right thing to do.